src/globalComponents.tsx

```diff
@@ -15,6 +16,9 @@ class GlobalStores {
 }
 
 (window as any).getStudySummaryUrl = getStudySummaryUrl;
+(window as any).oqlGenes = (oqlQuery:string)=>{
```

Comment:
@adamabeshouse Prolly not in this PR. But would be nice to expose these in a namespaced manner. E.g. `window.frontendVars.oqlGenes` similar for other variables/functions. Makes it a little easier to see in backend project where tight integration happens

Reply:
good idea